recipes/pept/meta.yaml

```
@@ -0,0 +1,72 @@
+{% set name = "pept" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: b044cc5b071d210816f9f7282b8330c17b3a624ff2bbc708b011ac06124663d3
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - cython >=0.29.14
+    - numpy >=1.18.5
```

(The `>=` will cause a problem with the way libraries are pinned from [`conda-forge-pinning-feedstock`](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml#L529-L532). As there is a version constraint it will ignore the global version pin. The `>=` will then result in the solver maximising the version. This will make it impossible to use this with anything other than the latest numpy.

If you can handle building against the globally pinned version (`1.14`) it will avoid a few potential issues that can happen:
```suggestion
    - numpy
```

If you need to use 1.18, then use. Though be aware that you might have to tweak this value from time to time to be consistent with the rest of the channel (not every version of numpy is rebuilt when things change, so when Python 3.9 is release things will fail and you'll have to use selectors to fix it):
```suggestion
    - numpy =1.18
```

---

The library can work with older versions of numpy - the 1.18.5 version was chosen mainly because:

1. It is the version available on Google Colab (the library will be used extensively there).
2. The new versions added a few `DeprecationWarning`s that we had to solve.

But it is not a requirement. Seeing that it is more of a headache to select that version, I'll delete the `>=`.
Thank you for your explanation!

---

Great. The solver will try to maximise the version number so the majority of people will get the latest `numpy` by default.